grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -495,5 +495,41 @@ class StockMovementController {
         }
     }
 
+    def exportPendingRequisitionItems = {
+        Location currentLocation = Location.get(session?.warehouse?.id)
+
+        def pendingRequisitionItems = stockMovementService.getPendingRequisitionItems(currentLocation)
+
+        def sw = new StringWriter()
+        def csv = new CSVWriter(sw, {
+            "Shipment Number" { it.shipmentNumber }
+            "Description" { it.description }
+            "Destination" { it.destination }
+            "Status" { it.status }
+            "Product Code" { it.productCode }
+            "Product" { it.productName }
+            "Qty Requested" { it.quantity }
+            "Qty Picked" { it.quantityPicked }
+            "Reason code" { it.reasonCode }
+        })
+        pendingRequisitionItems.each { requisitionItem ->
+            csv << [
+                    shipmentNumber  : requisitionItem?.requisition?.requestNumber,
+                    description     : requisitionItem?.requisition?.description ?: '',
+                    destination     : requisitionItem?.requisition?.destination,
```

Comment:
This works because the Location.toString() returns name, but it should be explicit in case we ever change the toString() implementation.
```
requisitionItem?.requisition?.destination?.name
```